### PR TITLE
[stable/jaeger-operator] Update image

### PR DESCRIPTION
**Pull request by naseemkullah**
#### What this PR does / why we need it:
Updates img to 1.10.0

#### Special notes for your reviewer:

cc @cpanato 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by naseemkullah** [comment]
cc @cpanato

**Comment by naseemkullah** [comment]
bump

File: stable/jaeger-operator/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.2.1
-appVersion: 1.9.0
+version: 2.2.2
```
Comment (cpanato) [inline stable/jaeger-operator/Chart.yaml]:
```suggestion
version: 2.3.0
```

**Review by cpanato — APPROVED** [review]

**Pull request merged**